rotkehlchen/chain/ethereum/modules/aave/v2/constants.py

```diff
@@ -8,3 +9,7 @@
     string_to_evm_address('0xEFFC18fC3b7eb8E676dac549E0c693ad50D1Ce31'),
     string_to_evm_address('0xcc9a0B7c43DC2a5F023Bb9b738E45B0Ef6B06E04'),
 )
+STK_AAVE_ADDR: Final = string_to_evm_address('0x4da27a545c0c5B758a6BA100e3a049001de870f5')
+A_STKAAVE: Final = Asset(f'eip155:1/erc20:{STK_AAVE_ADDR}')
```

Comment:
This is not needed. Just keep the STKAAVE_IDENTIFIER. No need to instantiate an asset and add it to the constants.